docs/content/topics/scripting/advanced.md

```diff
@@ -37,188 +47,207 @@ function exec(e, p) {
     .then(() => {
         console.log("done");
     });
```

Comment:
With the async/await example, you showed how to catch an error. Should we do the same here as well?

Reply:
I added a note to clarify that the same method can be used with either approach...